### PR TITLE
CM Flash Reco

### PR DESCRIPTION
**Pull request by bkimelman**
Changes to G4_TPC.C to include calls to new setting functions in CM Clusterizer and matcher.

### CODE REVIEW

File: detectors/sPHENIX/Fun4All_G4_sPHENIX.C [hunk 1]
```diff
@@ -49,7 +49,7 @@ int Fun4All_G4_sPHENIX(
     const string &outdir = ".")
 {
   Fun4AllServer *se = Fun4AllServer::instance();
-  se->Verbosity(0);
+  se->Verbosity(1);
```
[the remark this answers is not in the document]
Comment (osbornjd) [reply detectors/sPHENIX/Fun4All_G4_sPHENIX.C:52]: Once this gets fixed we can start a build with the coresoftware PR also to test everything with the CI

[the remark this answers is not in the document]
Comment (bkimelman) [reply detectors/sPHENIX/Fun4All_G4_sPHENIX.C:52]: Sorry about that, I must have missed it when merging my changes

**Comment by osbornjd** [comment]
Here's a build. You won't get an email when it completes, but it should be done by tomorrow morning US east coast time.

https://web.sdcc.bnl.gov/jenkins-sphenix/job/sPHENIX/job/Build-Master-gcc12/723/

File: detectors/sPHENIX/Fun4All_G4_sPHENIX.C (hunk 1 above)
[the remark this answers is not in the document]
Comment (osbornjd) [reply detectors/sPHENIX/Fun4All_G4_sPHENIX.C:52]: The fun4all verbosity values should be let alone, I suggest just checking out the version from the master branch and committing that on top of your current commit (or just change the Fun4AllServer verbosity back to 0 and the random seed verbosity back to 1)

**Comment by osbornjd** [comment]
Here's the latest build 
https://web.sdcc.bnl.gov/jenkins-sphenix/job/sPHENIX/job/Build-Master-gcc12/758/

**Pull request merged**